retroshare-gui/src/gui/MainWindow.h

```diff
@@ -324,6 +326,10 @@ private slots:
 
     QString nameAndLocation;
 
+    /** The menubar (Mac OS X only). */
```

Comment:
since this is MacOS code only, you should use the #ifdef here as well. No need to leave an uninitialized/unused pointer on other systems